pkg/inventory/inventorycm.go

```diff
@@ -94,7 +94,7 @@ func (icm *InventoryConfigMap) Load() (object.ObjMetadataSet, error) {
 // Store is an Inventory interface function implemented to store
 // the object metadata in the wrapped ConfigMap. Actual storing
 // happens in "GetObject".
-func (icm *InventoryConfigMap) Store(objMetas object.ObjMetadataSet) error {
+func (icm *InventoryConfigMap) Store(objMetas object.ObjMetadataSet, _ []ObjectStatus) error {
```

Comment:
We might want to serialize the status list into a string for the InventoryConfigMap. The current impl just has map keys, but we can uses the map values for status. Maybe serialize to a json string?

@seans3 what do you think?

Reply:
For the ConfigMap, I set the value of each key to the format of `{"actuation":XXX,"reconcile":XXX,"strategy":XXX}`. Let me know if this looks good to you.